src/components/ButtonsOnPetPage.tsx

```diff
@@ -1,16 +1,53 @@
+import { useState } from 'react';
```

Comment:
🟡  maybe rename `ButtonsOnPetPage` to` PetActionButtons` for better clarity and context.